Examples/CMakeLists.txt

```
@@ -129,6 +129,7 @@ function(add_warpx_test
         set(runtime_params
             "amrex.abort_on_unused_inputs = 1"
             "amrex.throw_exception = 1"
+            "amrex.signal_handling = 1"
```

Default.
```suggestion
            "amrex.throw_exception = 1"
```

More logic for both flags coming via: https://github.com/AMReX-Codes/amrex/pull/4165